### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Aniloren**
skeleton_for_bot creation for pull request

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+# pytype static type analyzer
+.pytype/
+
+# Cython debug symbols
+cython_debug/
+
+#IDEs
+.idea/
+.vscode
```
Comment (asyapolina) [inline .gitignore:130]: Github shows there is no blank line at the end of file, I think it would be good to add it

File: skeleton_for_bot.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import requests
+import json
+
+TOKEN = 'YOUR_TOKEN' # insert your token from BotFather here
```
Comment (asyapolina) [inline skeleton_for_bot.py]: I think `TOKEN` should be passed via command line arguments, we can use `import argparse`, add argument parsing function and start the application like this:
`python main.py -t <TOKEN> // --token <TOKEN>`

File: skeleton_for_bot.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+BASE_URL = f"https://api.telegram.org/bot{TOKEN}/"
+LONG_POLLING_TIMEOUT = 10
+
+def bot_function():
+    ...
```
Comment (asyapolina) [inline skeleton_for_bot.py]: If you want function that does nothing, specify `pass`  with appropriate comment in it's body - what this function should do in general.

File: skeleton_for_bot.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+
+def main(PLACEHOLDER_MSG="Привет") -> None:
```
Comment (asyapolina) [inline skeleton_for_bot.py]: I really think that `main` function shouldn't accept any arguments apart from command line ones. And type hinting here is redundant.

**Review by asyapolina — CHANGES_REQUESTED** [review]

**Comment by asyapolina** [comment]
Please, rename `skeleton_for_bot.py` to the `matrix_vision_bot.py`. And I agree that we should extend this implementation, at least add accepting/sending back the same image, i mean without processing it yet.

**Comment by Aniloren** [comment]
Ok

**Comment by Aniloren** [comment]
A little bit later

File: matrix_vision_bot.py.py [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+TOKEN = 'YOUR_TOKEN' # insert your token from BotFather here
+BASE_URL = f"https://api.telegram.org/bot{TOKEN}/"
+LONG_POLLING_TIMEOUT = 10
+
+def bot_function():
+    pass
+
+
+def main(PLACEHOLDER_MSG="Привет", any_arguments) -> None:
```
Comment (asyapolina) [inline matrix_vision_bot.py.py:12]: I meant smth like this, but ok. 
```suggestion
def main():
```
We will do the rest changes in separate PR. Approved.

**Review by asyapolina — APPROVED** [review]
